Libraries/Components/View/ViewAccessibility.js

```diff
@@ -46,6 +46,9 @@ export type AccessibilityRole =
   | 'toolbar'
   | 'grid';
 
+// Role types for web
+export type Role = 'slider' | 'img' | 'presentation' | 'summary';
```

Comment:
Missing roles that are already supported by accessibilityRole:

* alert
* button
* checkbox
* combobox
* link
* menu
* menubar
* menuitem
* none
* radio
* radiogroup
* scrollbar
* spinbutton
* switch
* tab
* tablist
* timer
* toolbar

Missing mapping:

* heading => header
* searchbox => search

Refer to [W3C roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) and [accessibilityRole](https://reactnative.dev/docs/view#accessibilityrole) docs.

I think all W3C roles (except "abstract" roles) should be allowed, since they might be used to provide the information for web targets, but those with no equivalent for native should be ignored.

Reply:
I have added all web equivalent native accessibility roles. Let me know if I have missed something.